website/content/api-docs/system/inspect/index.mdx

```diff
@@ -2,13 +2,18 @@
 layout: api
 page_title: /sys/internal/inspect - HTTP API
 description: >-
-  This endpoint is intended to inspect a specific internal subsystem for debugging purposes. It can be accessed with a root token or sudo privileges
+  This endpoint is intended to inspect a specific internal subsystem for debugging purposes, and turned off by default.
+  Once turned on, it can be accessed with a root token or sudo privileges
 ---
 
 # `/sys/internal/inspect`
 
 The `/sys/internal/inspect` family of endpoints is intended to inspect a specific internal subsystem for debugging purposes.
-It can be accessed with a root token or sudo privileges.
+This endpoint is off by default. See the
+[Vault configuration documentation](/docs/configuration) to
+enable. Once the endpoint is turned on, it can be accessed with a root token or sudo privileges.
+
+~> **NOTE**: These endpoints are only available in Vault version 1.13+. Backwards compatibility is not guaranteed. These endpoints are subject to change or may disappear without notice.
```

Comment:
Would you mind adding this note content to the router page as well? Anything we can do to make these harder to miss is a win.